Assets/Src/Scripts/SeedAvatar/UI/FpsViewport.cs

```diff
@@ -32,6 +33,11 @@ public class FpsViewport : MonoBehaviour {
     void Start() {
       _lastInterval = Time.realtimeSinceStartup;
       _frameCount = 0;
+      if (FpsLabel == null) {
```

Comment:
Hmm, this is not what I meant.

I was thinking the script `FpsViewport` can be added as a component to the `FpsText` game object in the scene.

In the script, you can then use
```
FpsLabel = GetComponent<Text>();
```


```
```